amanuensis/utils.py

```diff
@@ -11,19 +11,19 @@
 
 
 import bcrypt
-import boto3
 import flask
 import requests
 from cdislogging import get_logger
 import html2text
-from botocore.exceptions import ClientError
 from userportaldatamodel.driver import SQLAlchemyDriver
 from werkzeug.datastructures import ImmutableMultiDict
 
 from amanuensis.auth.auth import get_jwt_from_header
 from amanuensis.config import config
 from amanuensis.errors import NotFound, UserError
 
+from aws_client.boto import BotoManager
```

Comment:
You have already initialized BotoManager in the app __init__ so you can just call that instance instead of starting a second one. https://github.com/chicagopcdc/amanuensis/blob/8ebfc2fec04b1dbdeea43f1c8f52295d860394f0/amanuensis/__init__.py#L201

flask.app.boto, or flask.current_app.boto should do it